src/setup.js

```diff
@@ -3,38 +3,7 @@ import { LocalStorage } from './localstorage';
 const local = new LocalStorage();
 const session = new LocalStorage();
 
-global.localStorage = {
-  clear: jest.fn(() => local.clear()),
-  getItem: jest.fn(key => local.getItem(key)),
-  setItem: jest.fn((key, value) => local.setItem(key, value)),
-  removeItem: jest.fn(key => local.removeItem(key)),
-  key: jest.fn(index => local.key(index)),
-  toString: jest.fn(() => local.toString()),
-  get __STORE__() {
-    return local.store;
-  },
-  set __STORE__(store) {
-    local.store = store;
-  },
-  get length() {
-    return local.length;
-  },
-};
+global.localStorage = local;
+global.sessionStorage = session;
```

Comment:
For these I would probably just inline the `new LocalStorage()` calls instead of creating the `const` variables above.

Reply:
Yea, I agree